src/farm_designer/farm_events/farm_events.tsx

```diff
@@ -26,12 +26,14 @@ export class FarmEvents extends React.Component<FarmEventProps, {}> {
 
               let url = `/app/designer/farm_events/` +
                 (farmEvent.id || "UNSAVED_EVENT").toString();
-              let displayNameFields = farmEvent.displayName.split('\n');
-              let regimenName = null;
-              if (displayNameFields.length > 1) {
-                regimenName = <p style={{color:'gray'}}>{displayNameFields[0]}</p>;
+              let regimenName: JSX.Element;
+
+              if (farmEvent.subHeading) {
```

Comment:
I could have probably factored this down further into a `this.generateHeading()` helper or something like that, but should be good enough for now.